static/app/components/avatar/index.tsx

```diff
@@ -7,7 +7,7 @@ import TeamAvatar from 'sentry/components/avatar/teamAvatar';
 import UserAvatar from 'sentry/components/avatar/userAvatar';
 import {AvatarProject, AvatarSentryApp, OrganizationSummary, Team} from 'sentry/types';
 
-type Props = {
+export type AvatarProps = {
```

Comment:
Needed to export the type since I couldn't manage to get`React.ComponentProps`, `React.ComponentPropsWithRef` and React.`ComponentPropsWithoutRef` working properly. I think it might be something in how L25 of this file is declared.